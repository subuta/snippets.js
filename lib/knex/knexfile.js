import { build, format, snippets as s } from 'bld.js'

export default () => {
  return build`
    require('babel-register')
    module.exports = require('./src/api/utils/knex').config
  `
}
