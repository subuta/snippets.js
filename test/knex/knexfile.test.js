/* global expect, describe, it */

import { build, format, snippets as s } from 'bld.js'

import knexfile from 'lib/knex/knexfile'

describe('knex/knexfile', () => {
  it('should create knexfile', () => {
    expect(format(knexfile())).toMatchSnapshot()
  })
})
