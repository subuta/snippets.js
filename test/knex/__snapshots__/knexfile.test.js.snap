// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`knex/knexfile should create knexfile 1`] = `
"require('babel-register')
module.exports = require('./src/api/utils/knex').config
"
`;
